integration_tests/ads/test_ads.py

```diff
@@ -23,8 +23,8 @@ def test_create_ad_success(self):
         """
         ad = Ad.create(
             ad_account_id=DEFAULT_AD_ACCOUNT_ID,
-            ad_group_id=self.ad_group_utils.get_ad_group_id(),
-            creative_type="REGULAR",
+            ad_group_id=DEFAULT_AD_GROUP_ID,
+            creative_type="IDEA",
```

Comment:
REGULAR format pins are more common and will always stay. I think we should stick to creating an ad using that creative type instead of IDEA as it is a new creative type and we may see changes there.